### DISCUSSION TITLE
do i need to take note of balance before initiating a loop ?

### DISCUSSION THREAD

**Discussion by maximlomans** [Ride-The-Lightning/RTL#825] [Troubleshooting Questions] [answered]
**Is your feature request related to a problem? Please describe.**
maybe i should add beforehand,  i have the attention span of a fly and the memory of a banana stuffed, stoic monkey .. 

I have a Raspiblitz setup 1.7.0 

..every time i try to loop in our out, i end up having to close the process and reopen it, simply because the ui window covers the balance - i cant see what the balance is and it doesn't tell me it either,  at least not always... why i don't know. It was bad enough picking fights with autopilot over peers and channels ... damn thing has a monopoly on my funds : \ and peers
 now how did you guys miss that, 
or am i missing something ?
needless to add, it's annoying

**Comment by saubyk** [comment]
hey @marxenegls
can you please specify what device are you accessing RTL from? Is it a mobile?
Have you tried zooming out on the browser?

Also, it would be helpful if you have attach a screenshot to describe the problem you're referring to. Thanks.

**Comment by maximlomans** [comment]
I'm on [pi4] 4Gb -Raspiblitz but i do access it locally via my laptop screen res 1440 x 900 
.ex. whenever it's loop out it shows the remaining balance,  it doesn't do so every time - that's when i typically end up restart the process. If i got in the habit of taking note of the before i initiate loop .... 
maybe autopilot has something to do with it .
 It's not such a big deal, i suppose but it is inconvenient and mostly it's what i struggle with - so maybe i missed the best practises 

either way thanks for the swift reply ..
i reacted now impulsively after days of lamenting .

**Comment by maximlomans** [comment, marked as answer]
okay - i discovered that if spaces trail the sat then it wont show , if there aren't any it does 
'got it' 
 what discussion ? none here .

**Comment by maximlomans** [comment]


**Reply by saubyk** [reply]
😄
